executor/runtime/docker/docker_image_pull.go

```diff
@@ -88,6 +90,7 @@ func doDockerPull(ctx context.Context, cfg config.Config, metrics metrics.Report
 			for _, pullMessage := range pullMessages {
 				log.Warning("Pull Message: ", pullMessage)
 			}
+			log.Infof("Image pull took: %s", time.Since(pullStarted).String())
```

Comment:
go ahead and throw in the name of the image, since there will be multiple pulls per titus-executor